crates/sui-graphql-rpc/src/types/cursor.rs

```diff
@@ -180,6 +184,31 @@ impl<C> Page<C> {
     }
 }
 
+impl<C> Page<C>
+where
+    C: Checkpointed,
+{
+    // If cursors are provided, defer to the `checkpoint_viewed_at` in the cursor if they are
+    // consistent. Otherwise, use the value from the parameter, or set to None. This is so that
+    // paginated queries are consistent with the previous query that created the cursor.
```

Comment:
nit: doc comment.

```suggestion
    /// If cursors are provided, defer to the `checkpoint_viewed_at` in the cursor if they are
    /// consistent. Otherwise, use the value from the parameter, or set to None. This is so that
    /// paginated queries are consistent with the previous query that created the cursor.
```